docs/internal-documentation/maintenance/ReleasePlan.md

```diff
@@ -16,18 +17,21 @@ the respective "release schedule"
 The minor or major features that are part of each release are determined by the development team.
 
 ## Release schedule
-* a. Minor release schedule (TBD)
-* b. Major release schedule (TBD)
+The release schedule is to be created based on _minor releases_ of the Zonemaster parts, i.e. releases that changes then functionality but not the outward-facing API.
+
+When needed, _minor releases_ are changed to _major releases_ of one or more parts, i.e. when functionality changes require changes to the outward-facing API.
+
+When needed, _patch releases_ are published.
 
-The release schedule is determined by the development team. The release date is announce to the steering committee before "x" weeks. 
+When one or more parts are published as new release, the main Zonemaster repository is released listing the including components. At such event, there could be a mixture of major, minor and patch releases of the components.
 
```

Comment:
Maybe we should promise to provide patch releases for the previous major release until people have had reasonable time to upgrade to the new major release?